### PR TITLE
Make the module extension more user-friendly.

### PR DESCRIPTION
**Pull request by junyer**

### CODE REVIEW

**Comment by junyer** [comment]
PTAL, @rwgk and @meteorcloudy. :)

File: python_configure.bzl [hunk 1]
```diff
@@ -456,22 +456,13 @@ def _parse_my_own_version_from_module_dot_bazel(ctx):
...
-                      ))
-        elif module.tags.toolchain:
-            _fail("A non-root module may not specify any `toolchain` tags. " +
-                  "Found %r `toolchain` tags specified by module %r." % (
-                      len(module.tags.toolchain),
-                      module.name,
-                  ))
+        if not module.is_root or not module.tags.toolchain:
+            continue
+        toolchain = module.tags.toolchain[0]
```
Comment (meteorcloudy) [inline python_configure.bzl:461]: nit: I'm not completely sure if we need to check the size is 1?

**Review by meteorcloudy — APPROVED** [review]

File: python_configure.bzl (hunk 1 above)
Comment (junyer) [reply python_configure.bzl:461]: The control flow won't reach line 461 unless `module.tags.toolchain` contains at least one element.

**Comment by rwgk** [comment]
Thanks @meteorcloudy for the review!

**Pull request merged**